### PR TITLE
Update version to 2.11.11 with bug fixes

### PR DESCRIPTION
**Pull request by VipulMascarenhas**
### Description

This PR has code changes for the following:
- Update version and release notes.
- Integration test bug fix: 
     - Remove validation for byom FT model deployment and fine-tuning.
     - Update metadata to replace artifact location if already present.
     - Pop base custom model flag from FT model to ensure it is not listed when model_type is set as BASE.

### CODE REVIEW

**Review by darenr — APPROVED** [review]

**Review by mrDzurb — APPROVED** [review]

**Review by mayoor — DISMISSED** [review]

File: ads/aqua/common/utils.py [hunk 1]
```diff
@@ -668,23 +668,18 @@ def get_model_by_reference_paths(model_file_description: dict):
...
-            model["prefix"],
-        )
-        bucket_uri = f"oci://{bucket_name}@{namespace}/{prefix}".rstrip("/")
-        if bucket_name == AQUA_SERVICE_MODELS_BUCKET:
-            base_model_path = bucket_uri
-        else:
-            fine_tune_output_path = bucket_uri
+    if models:
+        if len(models) > 0:
+            base_model_artifact = models[0]
```
Comment (mayoor) [inline ads/aqua/common/utils.py:673]: Can you put a comment here why index the first model?

File: ads/aqua/common/utils.py [hunk 2]
```diff
@@ -668,23 +668,18 @@ def get_model_by_reference_paths(model_file_description: dict):
...
-            model["namespace"],
-            model["bucketName"],
-            model["prefix"],
-        )
-        bucket_uri = f"oci://{bucket_name}@{namespace}/{prefix}".rstrip("/")
-        if bucket_name == AQUA_SERVICE_MODELS_BUCKET:
-            base_model_path = bucket_uri
-        else:
-            fine_tune_output_path = bucket_uri
+    if models:
```
Comment (mrDzurb) [inline ads/aqua/common/utils.py:671]: I guess we can simplify a logic a bit here, something like: 

```
if not models:
     rise AquaValueError
 if len(models)>0:
      base_model_artifact = models[0]
 if len(models)>1:
        ft_model_artifact = models[1]
```

**Review by mrDzurb — APPROVED** [review]

File: ads/aqua/common/utils.py (hunk 2 above)
Comment (VipulMascarenhas) [reply ads/aqua/common/utils.py:671]: yes, will simplify this. I have trouble sleeping when we raise AquaValueError anywhere now.

Comment (VipulMascarenhas) [reply ads/aqua/common/utils.py]: updated.

File: ads/aqua/common/utils.py (hunk 1 above)
Comment (VipulMascarenhas) [reply ads/aqua/common/utils.py]: updated.

**Review by mrDzurb — APPROVED** [review]

**Review by mayoor — APPROVED** [review]

**Pull request merged**